terraform-infra-approvals/global.json

```diff
@@ -26,6 +26,7 @@
     {"source":  "git@github.com:hmcts/cnp-module-app-service-plan?ref=master"},
     {"source":  "git@github.com:hmcts/cnp-module-waf?ref=master"},
     {"source":  "git@github.com:hmcts/cnp-module-trafficmanager-endpoint?ref=master"},
-    {"source":  "git@github.com:hmcts/cnp-module-metric-alert?ref=master"}
+    {"source":  "git@github.com:hmcts/cnp-module-metric-alert?ref=master"},
+    {"source":  "git@github.com:hmcts/moj-module-webapp?ref=master"}
```

Comment:
that module doesn't exist anymore, it redirects to cnp-module-webapp, please update your config